docs/packaging.rst

```diff
@@ -2,5 +2,89 @@
 Packaging Guide
 ===============
 
-Documentation can be found at
-https://github.com/letsencrypt/letsencrypt/wiki/Packaging.
+Releases
+========
+
+We release packages and upload them to PyPI (wheels and source tarballs).
+
+- https://pypi.python.org/pypi/acme
+- https://pypi.python.org/pypi/letsencrypt
+- https://pypi.python.org/pypi/letsencrypt-apache
+- https://pypi.python.org/pypi/letsencrypt-nginx
+- https://pypi.python.org/pypi/letshelp-letsencrypt
+
+The following scripts are used in the process:
+
+- https://github.com/letsencrypt/letsencrypt/blob/master/tools/release.sh
+- https://gist.github.com/kuba/b9a3a2ca3bd35b8368ef
+
+We currently version as ``0.0.0.devYYYYMMDD``, and will change at GA time to the following scheme:
+
+- ``0.1.0``
+- ``0.2.0dev`` for developement in ``master``
+- ``0.2.0`` (only temporarily in ``master``)
```

Comment:
GA happened back in December and we now use the versioning scheme described here rather than `0.0.0.devYYYYMMDD`.